Tests/RxCocoaTests/SharedSequence+ConcurrencyTests.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  SharedSequence+ConcurrencyTests.swift
+//  Tests
+//
+//  Created by Shai Mishali on 22/09/2021.
+//  Copyright © 2021 Krunoslav Zaher. All rights reserved.
+//
+
+#if swift(>=5.5.2) && canImport(_Concurrency) && !os(Linux)
+import Dispatch
+import RxSwift
+import RxCocoa
+import XCTest
+import RxTest
+
+@available(macOS 12, iOS 15, tvOS 15, watchOS 8, *)
```

Comment:
I guess platform availibility should be updated in tests as well:
`@available(macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0, *)`

Reply:
No, I'm using `.values` there which is only available on iOS 15.

Reply:
Are you sure? As far as I can tell you've made it available in iOS 13 in c03353a

Reply:
I'll check a bit later, thanks :)